anu-001/FirstForFood
# #160641480 Project Setup for Persistence Database

#### What does this PR do?
Have the project folder fashioned with MVC architecture, and setup databases for development and testing

#### Description of Task to be completed?
Setup database for persistence data

#### How should this be manually tested?
Clone the repository and change directory in to it, run npm install
Change directory to api/v2/, there you will see the config folder, that contains connection and config files. The databases URI are hidden in .env file, as environment variables.
       
#### Any background context you want to provide?
This implementation use cloud implementation of Postgresql database: Heroku and ElephantSQL

#### What are the relevant pivotal tracker stories?
[160641480](https://www.pivotaltracker.com/story/show/160641480)
